### DISCUSSION TITLE
All Spiral Current Releases

### DISCUSSION THREAD

**Discussion by ghost** [SpiralLinux/SpiralLinux-project#319] [General]
ok, first off Thank You for creating all these different spins  :)   I apperciate it :)

second, I have MS so my 40 years in IT are prolly worthless but I wanted to post my findings so you could think about if they should be fixed.

I installed, or tried in some cases, in various HDD configurations them all on my Frankenstein Laptop.

Test 1 was with XFCE, I have a broken main monitor so getting to Display settings was an important 1st step.  It works reasonably well.

Test 2 was with Budgie, same issue with Display but I was able to finally get the HDMI working and Budgie worked faster than XFCE

Test 3 was with Cinnimon (spell?) and accessing the Display setting was very easy and performance was about like XFCE

Test 4 was with Mate and accessing Display settings wasnt horrible ... performance was like XFCE also

Test 5 was with Lxtq(spell?) and I could not even attempt to use as there was no way to change the display...at least I could not do it  👎  so it wasnt tested...

Test 6 was with KDE Plasma and while it looks great it was the slowest of all the spins  👎   It was tested just a bit...

Test 7 was with the 'Build' spin.  I guess it could be a repair spin?  Not sure of the point but it must be for truly geekie Linux peeps?  And it was the fastest but in my mind from a usability perspective sort of pointless  ?  who knows...

In summary the Display Setting should be readily accessable before install can even be considered.

I used my Intel i5 with 8GB ram.  The "Faster than a Snail" was a great Logo and the cp'ing 4TB of data was tried/attempted in all the above tests...for the most part it went well BUT the KDE setup doesnt like releasing cp file action related cmds and that prolly needs to be looked at.

Hey, at least I didnt pull out my E1 Gateway Vision to do the tests  :) 

lol/Bill :)

**Comment by geckolinux** [comment]
Hi there Bill, thanks a lot for reporting your experiences and for the detailed testing. It's true that not all SpiralLinux editions are the same, and some probably aren't ideal for certain types of hardware and usage scenarios. I hope you find an edition that suits your needs and preferences. Thanks again!

**Comment by ghost** [comment]
KDE is not horrible, it is just bad  :P ...
